internal/security/secretstore/init.go

```diff
@@ -340,36 +340,45 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		os.Exit(1)
 	}
 
-	cert := NewCerts(req, configuration.SecretService.CertPath, rootToken, configuration.SecretService.GetSecretSvcBaseURL(), lc)
-	existing, err := cert.AlreadyinStore()
-	if err != nil {
-		lc.Error(err.Error())
-		os.Exit(1)
-	}
+	// Was CertPath/CertFilePath/KeyFilePath set in configuration.toml for kong/proxy?
+	if configuration.SecretService.CertPath != "" && configuration.SecretService.CertFilePath != "" && configuration.SecretService.KeyFilePath != "" {
```

Comment:
would suggest trimming spaces to be safer

Reply:
Done

Reply:
didn't see it

Reply:
Crap - I read your comment wrong...let me trim it.

Reply:
Shouldn't this be handled on the read in of the configuration.toml anyway? I'm looking for that first...

Reply:
@jim-wang-intel let's discuss at stand-up - I feel like this should be handled further upstream like in the config unmarshalling instead of here.